src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunction.cs

```diff
@@ -32,6 +32,7 @@
 using Microsoft.PowerFx.Syntax;
 using Microsoft.PowerFx.Types;
 using static Microsoft.PowerFx.Core.IR.IRTranslator;
+using static Microsoft.PowerFx.Syntax.PrettyPrintVisitor;
 using CallNode = Microsoft.PowerFx.Syntax.CallNode;
```

Comment:
unused?

Reply:
I have no idea why VS keeps inserting that.  Removed.